.gitattributes

```diff
@@ -1,4 +1,3 @@
-/tests export-ignore
```

Comment:
I've reverted the change and I'll fully remove shallow_clone to fix the underlying issue. Thanks for the tip.